components/SiteNavbar.vue

```diff
@@ -83,29 +83,38 @@
   </div>
 </template>
 
-<script>
-export default {
-  data() {
-    return {
-      lg: false,
-      mobileNavIsActive: false,
-    };
-  },
+<script lang="ts">
+import Vue from 'vue'
+
+/**
+ * TODO use https://vueuse.org/core/usebreakpoints and
+ * https://vueuse.org/core/useEventListener/
+ */
+
+export default Vue.extend({
+  data: () => ({
+    lg: false,
+    mobileNavIsActive: false,
+  }),
 
   /* Need to disable the `sort-keys` rule here, since it
    * conflicts with the `vue/order-in-components` rule.
    */
   /* eslint-disable sort-keys */
```

Comment:
About that one, it bothers me a lot: could you try and see if we can use this rule instead of sort-keys? https://eslint.vuejs.org/rules/sort-keys.html